src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -10,13 +10,14 @@
         <service id="controller_name_converter" class="Symfony\Bundle\FrameworkBundle\Controller\ControllerNameParser">
```

Comment:
would it make sense to give this service a different name, then create a deprecated alias with this name, to warn ppl that use this id for referencing the service?

Reply:
👍 done